src/pipeline.cc

```diff
@@ -1199,6 +1200,25 @@ NAN_METHOD(pipeline) {
   baton->webpLossless = AttrTo<bool>(options, "webpLossless");
   baton->webpNearLossless = AttrTo<bool>(options, "webpNearLossless");
   baton->tiffQuality = AttrTo<uint32_t>(options, "tiffQuality");
+  // tiff compression options
+  std::string tiffCompression = AttrAsStr(options, "tiffCompression");
```

Comment:
Given we're going for a string-based input here, can we use `vips_enum_from_nick` for `compression` and `predictor`? (like the [kernel](https://github.com/lovell/sharp/blob/72f69dda30e5b81d24e89e0ac5fdd2d9a9a7f5cc/src/pipeline.cc#L394) property of the resize operation.)